recipe/meta.yaml

```diff
@@ -61,4 +61,3 @@ extra:
   recipe-maintainers:
     - klauer
     - zllentz
-    - zrylettc
```

Comment:
```suggestion
    - tangkong
```

Reply:
Interestingly, you can't commit suggestions from the github ui on deleted lines. TIL.